cfgov/housing_counselor/jinja2/housing_counselor/index.html

```diff
@@ -47,29 +47,57 @@ <h1>Housing counselors near you</h1>
+                            <div class="o-sidebar-breakout_col
+                                        content-l_col
+                                        content-l_col-2-3">    
+                                <h1>Find a housing counselor</h1>
+                                <p>
+                                    Housing counselors throughout the country can
+                                    provide advice on buying a home, renting,
+                                    defaults, foreclosures, and credit issues.
+                                    Using the search box below,
+                                    you can find one near you.
+                                    The counseling agencies on this list are
+                                    approved by the U.S. Department of Housing
+                                    and Urban Development (HUD) and they can
+                                    offer independent advice about whether a
+                                    particular set of mortgage loan terms is a
+                                    good fit based on your objectives and
+                                    circumstances, often at little
+                                    or no cost to you. This list will show you
+                                    several approved agencies in your area.
+                                    There is also a
+                                    <a class="a-link a-link__icon"
+                                       href="https://apps.hud.gov/offices/hsg/sfh/hcc/hcs.cfm">
+                                        <span class="a-link_text">list of nationwide HUD-approved counseling agencies</span>
+                                        {{ svg_icon( 'external-link' ) }}</a>.
+                                </p>
+                            </div>
+                            <aside class="o-sidebar-breakout_col
+                                          content-l_col
+                                          content-l_col-1-3">
+                                <header class="m-slug-header">
+                                    <h2 class="a-heading">
+                                        Coronavirus affecting your mortgage or housing?
+                                    </h2>
+                                </header>
+                                <div class="o-sidebar-breakout_text-container">
+                                    <div class="o-sidebar-breakout_text-body">
+                                        <p>
+                                            We have resources from multiple federal agencies to help homeowners and renters understand options for relief and protection in light of the coronavirus emergency.
+                                        </p>
+                                        <p>
+                                            <a href="https://www.consumerfinance.gov/coronavirus/mortgage-and-housing-assistance/">
```

Comment:
```suggestion
                                            <a href=/coronavirus/mortgage-and-housing-assistance/">
```

Could we make this a relative URL so that it works properly on non-production deployments?

Reply:
Fixed in https://github.com/cfpb/consumerfinance.gov/pull/6249/commits/3bf444ec17339eede551a24da7e781398d6ea85b